Kinvey-Xamarin/Kinvey-Xamarin.csproj

```diff
@@ -175,6 +183,12 @@
     <Reference Include="Newtonsoft.Json">
       <HintPath>..\packages\Newtonsoft.Json.8.0.2\lib\portable-net45+wp80+win8+wpa81+dnxcore50\Newtonsoft.Json.dll</HintPath>
     </Reference>
+    <Reference Include="crypto">
+      <HintPath>..\packages\Portable.BouncyCastle-Signed.1.7.0.2\lib\portable-net45+dnxcore50+wp80+win+wpa81+MonoTouch10+MonoAndroid10+xamarinmac20+xamarinios10\crypto.dll</HintPath>

```

Comment:
That is correct.  [BouncyCastle](https://www.nuget.org/packages/Portable.BouncyCastle/) is a cryptography package required by PubNub.